### DISCUSSION TITLE
Can I include form data in new logs?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#3960] [Q&A] [answered]
In order to investigate why some users have trouble creating data, I'm considering logging their form data. This would allow me to identify any errors or missing information and help resolve the issue faster, ultimately improving the user experience.

**Comment by ganigeorgiev** [comment, marked as answer]
The log attributes can be practically anything as long as the value can be serialized to something useful.

If you haven't already, you can also check:

- Go: https://pocketbase.io/docs/go-logging/
- JS: https://pocketbase.io/docs/js-logging/

**Reply by hungcrush** [reply]
Thank you!!!!! I got it.
